source/_docs/configuration/templating.markdown

```diff
@@ -315,40 +315,41 @@ Closest to some entity:
 
 ### Formatting
 
-- `float` will format the output as float.
+- Global function `float` will attempt to convert given value to float.
```

Comment:
I don't think that we should start using `global` because without additional details it's not clear what it means. E.g., difference to `local` functions.

Reply:
Well, I wanted to highlight the fact that it's not a function of a particular class to avoid interpretations like "calling my_var.float() is fine".
Do you think it's important here or it's better to just drop "global"?